component/file_cache/file_cache.go

```diff
@@ -337,7 +337,7 @@ func (c *FileCache) StatFs() (*syscall.Statfs_t, bool, error) {
 	statfs := &syscall.Statfs_t{}
 	err := syscall.Statfs("/", statfs)
 	if err != nil {
-		log.Debug("FileCache::StatFs : statfs err [%s].", err.Error())
+		log.Debug("FileCache: statfs err [%s].", err.Error())
```

Comment:
For any component level logs its always "<component>::<function> : " as prefix so existing log is correct as per our practices.